src/style.css

```diff
@@ -1,3 +1,82 @@
+:root {
+  --light-blue: #00acdc;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, Arial, serif;
+  font-size: 12px;
+  font-weight: 500;
+  font-style: normal;
+}
+
+.header {
+  box-sizing: border-box;
+  padding-left: 50px;
+  padding-right: 50px;
+  width: 100%;
+  height: 60px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  justify-content: space-between;
+}
+
+.header-logo {
+  display: block;
+}
+
+.nav__list {
+  padding-inline-start: 0;
+  display: inline;
+}
+
+.nav__item {
+  display: inline-block;
+}
+
+/* a {
+  
+} */
+
+.links {
+  width: 636px;
+  text-align: right;
+  height: 60px;
+  line-height: 60px;
```

Comment:
Don't fix container size (if there is no such requirement). The width of '.links' is fixed to 636px, which may not be flexible for different content sizes or screen widths. Consider using a more responsive approach unless there is a specific requirement to fix the size.